### ISSUE TITLE
Experimental pragma generates warnings

### ISSUE THREAD

**Issue by elenadimitrova** [duaraghav8/Ethlint#201]
Since [solc 0.4.21](https://github.com/ethereum/solidity/releases/tag/v0.4.21) `pragma experimental` no longer generates solc compiler warnings while with `solium 1.1.7` it still does:
```
warning    Avoid using experimental features in production code    no-experimental
```

**Comment by duaraghav8** [comment]
@elenadimitrova this is intentional. The rule itself is also highly debatable, since using experimental features goes against good practices but some contracts can simply not function without them so they have no choice but to use them.
If you'd like to avoid this warning, please disable `no-experimental` (either in soliumrc or through comments).
Hope that clarifies?

**Comment by duaraghav8** [comment]
I also just realized that I forgot to mention in the release notes that `no-experimental` was released in v1.1.7.

**Comment by elenadimitrova** [comment]
I would have agreed with this before `solc@0.4.21` but those release notes will be contradicting with `solium` output, namely:
>In this release, this pragma does not generate a warning anymore, so you can (and should) use it in production code.

**Comment by duaraghav8** [comment]
ah, I see. Since the release itself recommends its use in prod, I'll modify the rule to not flag experimental pragmas.

**Comment by duaraghav8** [comment]
already fixed and released, forgot to close issue before.
